.github/workflows/rhel7.yml

```diff
@@ -0,0 +1,126 @@
+name: RHEL 7 x86_64
+on:
+  schedule:
+    - cron: "0 1 * * *"
+  workflow_dispatch: {}
+env:
+  BTFHUB_NO_SUDO: true
+jobs:
+  rhel7-update:
+    name: rhel 7 x86_64
```

Comment:
let's make a convention for job name across all workflows

Reply:
This is my preferred convention, but I don't want to change all the other jobs in this PR.